_build_cfg.yml

```diff
@@ -179,6 +177,9 @@ Topics:
     Distros: openshift-enterprise
   - Name: Master and Node Configuration
     File: master_node_configuration
+  - Name: Image Streams and Templates
+    File: imagestreams_templates
+    Distros: openshift-enterprise,openshift-origin
```

Comment:
I don't think we need a Distros here since the whole book is already Distro'd for only Origin + Enterprise.